### ISSUE TITLE
IPR027238 not all DNA helicases

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#3842]
* **InterPro ID / label**

GO:0043139 | 5'-3' DNA helicase activity | IEA with IPR027238

* **Example sequences with problematic annotation (ID + gene/protein name):**

rvb1 https://www.pombase.org/gene/SPAPB8E5.09
rvb2 https://www.pombase.org/gene/SPBC83.08

* **Description of issue**
not all  proteins with helicase domains have helicase activities in vivo (the helicase name is a bit of a misnomer)

**Comment by ValWood** [comment]
Actually, I did not look in detail, but you can probably map this sub family to 
ATP-dependent chromatin remodeler activity (GO:0140658)

**Comment by EBI-Hsinyu** [comment]
Hi Val, 

Thank you for the e-mail. I found several members in this entry have helicase activities (papers listed below). One paper states "Rvb1 and Rvb2 are highly conserved, essential AAA+ helicases found in a wide range of eukaryotes" (PMID: 20130677). Of course we can't be certain that all members have this activity. However, InterPro tries to predict this from the annotated members (as we usually do). Having said that, we can perhaps go for a less specific "GO:0003678 DNA helicase activity". Will this be ok?

TIP49b, a new RuvB-like DNA helicase, is included in a complex together with another RuvB-like DNA helicase, TIP49a (PMID: 10428817)
Yeast Rvb1 and Rvb2 are ATP-dependent DNA helicases that form a heterohexameric complex (PMID: 18234224)

Hsin-Yu

**Comment by ValWood** [comment]
Hi Hsin-Yu, these are old papers. 
See also this analogous ticket:
https://github.com/geneontology/go-ontology/issues/21776#issuecomment-874773671

@colinlog

**Comment by colinlog** [comment]
PMID:20130677 is a review, just propagating the possibly erroneous statement  'RuvB-like 1/2 are helicases', not demonstrating helicase activity

PMID:10428817 shows helicase activity by the purified Tip49b and that it bind Tip49a, but no helicase assays were performed with reconstituted Tip49a + Tip49b.

PMID:10428817 also refers to PMID:10336418 and there the rat protein Tip49a was shown to be an ATPase that is best stimulated by single strand (M13) DNA, and to have 3' to 5' helicase activity. This is not done with the Tip49a+b complex. The reactions involve much less substrate (10 ng of M13+annealed oligos) than enzyme (200ng) and are therefore not optimal/catalytic even. In 2020, the Rvb1/2 are said to be weak ATPases and it is not metioned anywhere that they have helicase activity (PMID:33205750). 

[The paper to look at for Rvb1/2 (Tip49a/b) helicase activity (and hexameric ring formation!) is PMID:18234224. What puzzles me is that in that paper the 5' to 3' helicase activity is strongest whilst in the other papers (including the Nature paper from 2000 PMID:10952318 as well as in PMID:10428817 and PMID:10336418 it was 3' to 5' activity that was highlighted and 5' to 3' that was considered weak. For sure, in PMID:18234224 the 3' to 5' helicase activity is weaker, and neither the Tip49 a only nor the Tip49b only reactions show any helicase activity.

All in all, I am still unsure what helicase activity to annotate, but from this paper (PMID:18234224), the Rvb1/Rvb2 (aka Tip49a/b) complexes would be 5' to 3' helicases.] [Note that this last is a copy-paste from my comment on ticket #21776]

**Comment by EBI-Hsinyu** [comment]
Hi Val, 
I am not quite sure we can ignore these old papers (PMID:10428817, PMID:18234224) though. The new paper (PMID:33205750) didn't show the evidence that they are not helicases, it only showed that their ATPase activity is regulated by their interaction with DHX34. Just a thought.

Hsin-Yu

**Comment by EBI-Hsinyu** [comment]
Sorry just saw the Colin's reply. Would it be fair to add just the "GO:0003678 DNA helicase activity" or not?

**Comment by ValWood** [comment]
I think the safest annotation would be
ATPase, acting on DNA (GO:0008094).

Once we can see if any families really are only 
ATP-dependent chromatin remodeler activity (e.g. fft3, hrp1, hrp3, mit1, snf22 - sorry I don't know all the human names)
OR
DNA helicase activity (GO:0003678)  (fml1, fml2).

The mappings can be made more specific.

It really seems that  current thinking is that the  _in vitro_ role of the chromatin remodelling ATPases is not  a helicase.  I went through the pombe ones and I could not find any good evidence to support helicase annotation.
Usually they say "ATPase, helicase family" and we annotated incorrectly  to 'helicase'. It would be useful if the super-families could be renamed to "ATPase acting on DNA".

This is reminiscent to the ATP transporter domain, which continually get annotated to transmembrane transporter/transport, but are  found in lots of other contexts.
